pkg/client/keycloak/client.go

```diff
@@ -175,6 +175,17 @@ func (kc *kcClient) GetToken() (string, error) {
 	return tokenResp.AccessToken, nil
 }
 
+// IsJWTTokenExpired returns false if JWT token is not expired, otherwise returns false
+// This method does NOT validate the token, and should not be used when validation is necessary
+func IsJWTTokenExpired(accessToken string) bool {
+	if token, _ := jwt.Parse(accessToken, nil); token != nil {
+		tokenClaims := token.Claims.(jwt.MapClaims)
+		exp := tokenClaims["exp"].(float64)
```

Comment:
You can follow this to get expiry in time format and using time before method to return true or false: https://stackoverflow.com/questions/58441574/how-to-parse-the-expiration-date-of-a-jwt-to-a-time-time-in-go

Reply:
Brilliant! Thanks @Rajagopalan-Ranganathan 
I've added that adjustment. Do you think I should separate `exp` into two steps or is it ok?